CONTRIBUTING.md

```diff
@@ -1,19 +1,59 @@
 ## Welcome!
 
-We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+We’re so glad you’re thinking about contributing to an 18F open source project! If you’re unsure about something, feel free to submit your issue or pull request anyway. The worst that can happen is you’ll be politely asked to change something. We love all friendly contributions and questions.
 
 We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
 
-We encourage you to read this project's CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md).
+We encourage you to read this project’s CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md).
 
 If you have any questions or want to read more, check out the [18F Open Source Policy GitHub repository]( https://github.com/18f/open-source-policy), or just [shoot us an email](mailto:18f@gsa.gov).
 
+## How to contribute
+
+### Where to submit a pull request
+
+Submit pull requests to the `18f-pages-staging` branch. This is the main branch of Methods. If you’re not sure how to submit a pull request, feel free to submit an issue and we will try to address it ourselves.
+
+### What to include in a pull request
+
+Including these will help us be faster and better at responding.
+
+- **Changes:** a list, preferably bulleted, of what you’re changing
+- **Notes:** anything that might help us understand the change, like why you’re doing it or alternative approaches you tried
+- **Screenshots:** if your pull request will affect layout or interaction, to show us what you’re fixing or changing
+
+### What to include in an issue
+
+Not all of these will apply to all issues. But when they do, it’s very helpful to see them (special thanks to @ondrae for [the suggestion of this list](https://github.com/18F/open-source-guide/issues/15#issuecomment-129552978)).
```

Comment:
@colinpmacarthur That definitely needs to be clear. Is there something in the existing copy that implies they are all bugs? Or is that a default assumption for a lot of people and we just need to be more explicit about it?